web/src/components/Trade/index.js

```diff
@@ -143,6 +143,10 @@ const SendPage = ({setPage}) => {
   const handleCancel = () => setPage(0);
   const handleBack = () => {
     const num = step - 1 > 0 ? step : 0;
```

Comment:
Optional, I think we use function handleBack() only in step 1. 
The better way is to use the below codes to replace lines (145~150). Thanks.
```
setIsFixedAmount(false);
setSendPoint(0);
setStep(0);
```

Reply:
@gcobcqwe Updated, thanks!